.github/workflows/seo.yml

```diff
@@ -0,0 +1,77 @@
+---
+name: SEO CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v1.8.0
+        with:
+          fail: true
+          args: --max-redirects 10 -a 403,500,503 .
+  seo_spy_orphan_pages:
+    runs-on: ubuntu-latest
+    name: Check orphan pages
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Checkout SEO Spy
+        uses: actions/checkout@v3
+        with:
+          repository: 3mdeb/seo-spy
+          path: seo-spy
+      - uses: actions/setup-python@v3
+        with:
+          python-version: '3.9'
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r seo-spy/requirements.txt
+          pip install -r requirements.txt
+      - name: Serve site
+        run: |
+          mkdocs serve &
+          sleep 15
+      - name: Check orphan pages
+        uses: 3mdeb/seo-spy-action@main
+        with:
+          domain: 'http://127.0.0.1:8000'
+  seo_spy_canonical_links:
```

Comment:
@artur-rs Paradoxically, this will increase the execution time of CI. Currently each job (link checker, orphan pages, canonical) runs in parallel. Merging orphan pages and canonical will create a dependency, forcing one process to wait for the other to finish before proceeding. This would also result in less readability and the need to scroll to get the result of the first job.

Reply:
We could still build once and ruse the content of `static` in all jobs. We already have to build site files in: https://github.com/Dasharo/docs/blob/master/.github/workflows/deploy.yml
But if it it fast enough maybe we do not care.

Reply:
@artur-rs I've created discussion: https://github.com/Dasharo/docs/discussions/627#discussion-5472684

Reply:
Thanks